code/datums/uplink_item.dm

```diff
@@ -248,17 +248,26 @@ var/list/uplink_items = list()
 /datum/uplink_item/dangerous/syndieborg
 	name = "Syndicate Cyborg"
 	desc = "A cyborg designed and programmed for systematic extermination of non-Syndicate personnel."
-	item = /obj/item/weapon/antag_spawner/borg_tele
+	item = /obj/item/weapon/antag_spawner/nuke_ops/borg_tele
 	cost = 50
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 0
 
 //for refunding the syndieborg teleporter
 /datum/uplink_item/dangerous/syndieborg/spawn_item()
-	var/obj/item/weapon/antag_spawner/borg_tele/T = ..()
+	var/obj/item/weapon/antag_spawner/nuke_ops/T = ..()
 	if(istype(T))
 		T.TC_cost = cost
 
+
+/datum/uplink_item/dangerous/reinforcement
+	name = "Reinforcements"
+	desc = "Call in an additional team member. They won't come with any gear, so you'll have to save some telecrystals to arm them as well."
+	item = /obj/item/weapon/antag_spawner/nuke_ops
+	cost = 25
+	gamemodes = list(/datum/game_mode/nuclear)
+	surplus = 0
+
```

Comment:
Should probably do the same `TC_cost` shebang as above. Might make sense to make it a child.